web/app/datasets/handle_signals.py

```diff
@@ -113,5 +124,12 @@ def handle_signals():
     """
     Entry point (called via manage.py), retrieve and handle signals.
     """
-    for signals in _batch_signals():
+    # FIXME: for now only acceptance is supported (and hardcoded below)
+    acceptance = True
+    email = os.getenv('SIGNALS_USER', 'signals.admin@amsterdam.nl')
```

Comment:
Deze config zou ik buiten de code houden (meer docker config)

Reply:
(Zie onderaan voor een voorgestelde fix.)

Ik wil graag duidelijke foutmeldingen op de plek waar de fout wordt geconstateerd, dus ik blijf lokaal een exceptie raisen. Dit is handig tijdens het ontwikkelen, en voor het opsporen van problemen in productie. (En deze exceptie gebeurd niet in de service, maar in het periodiek aangeroepen script.) 

Als je vasthoud aan een centrale check, dan is het health check endpoint volgens mij de goede plek om het in te bouwen (controleren of alle env variabelen zijn gezet - mogelijk configureerbaar). Dan kan het productie systeem niet "up" komen (health check faalt bij misconfiguratie) en lokaal kun je de service draaien zonder credentials (ook voor tests).

Terzijde: een centrale check vangt ook niet alle problemen rond connecties af (Services die down zijn, of verkeerde credentials komen gewoon door de centrale checks en zorgen voor problemen op de zelfde plek in de code waar ik nu excepties raise.)

**Voorstel:**
* Health check updaten met een check voor het aanwezig zijn de benodigde env variabelen (resultaat, service en scripts crashen niet door afwezige credentials - systeem monitoring ziet dat de service niet start als ze ontbreken).
* Lokaal foutmeldingen genereren (zodat je aan de code kunt werken zonder alle credentials van alle externe services). Is een achtervanger voor ontbrekende credentials, en niet heel anders dan de andere excepties die bij het connecten met externe services kunnen gebeuren (maar explicieter).

Reply:
Snelle update, n.a.v. gesprek:

* centrale configuratie
* health check
* lokaal kunnen draaien zonder alle credentials